tests/data/bundles/test_yahoo.py

```diff
@@ -1,11 +1,13 @@
+from __future__ import division
```

Comment:
Just a note: before merging the PR, we should clean this up.  Ideally. we shouldn't be seeing unrelated changes from other PRs.